types/container/host_config.go

```diff
@@ -216,6 +216,7 @@ type HostConfig struct {
 	GroupAdd        []string           // List of additional groups that the container process will run as
 	IpcMode         IpcMode            // IPC namespace to use for the container
 	Links           []string           // List of links (in the name:alias form)
+	Aliases         []string           // List of aliases
```

Comment:
I wonder if we should make this name more specific to the type of aliases?  

`HostnameAliases` maybe ?